client/src/app/site/pages/organization/organization-routing.module.ts

```diff
@@ -41,6 +41,12 @@ const routes: Routes = [
                 data: { omlPermissions: [OML.can_manage_organization] },
                 canLoad: [PermissionGuard]
             },
+            {
+                path: `mediafiles`,
+                loadChildren: () => import(`./pages/mediafiles/mediafiles.module`).then(m => m.MediafilesModule),
+                data: { omlPermissions: [OML.can_manage_organization] },
```

Comment:
I think it was intended that all users could access the files, or do I remember it wrong and only the _files_ themselves should be accessible, and not the whole tree?

Reply:
This should disable the *entire* file page for all users except from orga-admins (i.e. the base-level files can't be seen by anyone else either). This setting is there because I didn't know who was intended to have access to this so I asked @emanuelschuetze  and he told me to just restrict access to orga-admins for now (since it currently does not have any special features (like f.E. transferring files to specific meetings) anyway). This is meant to be temporary AFAIK.

Reply:
ok sure, we can still change it later